examples/salesforce/index.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const jsforce = require('jsforce');
+const everyauth = require('../..');
```

Comment:
Since this is an EveryAuth example, let's not assume this runs under the everyauth-express project, User is probably following this example in his own Express App, so use `const everyauth = require('@fusebit/everyauth-express');` instead.

Reply:
I'd actually expect them to run it in the context of the cloned repo - this is the simplest way to get the code of the sample.

Reply:
Changed to fixed dep